android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java

```diff
@@ -57,4 +58,8 @@ protected int update(String tableName, ContentValues values,
     protected Cursor rawQuery(String sql, String[] selectionArgs) {
         return mDb.rawQuery(sql, selectionArgs);
     }
+
+    protected SQLiteStatement compileStatement(String sql) {
```

Comment:
In a followup PR, can we annotate method parameters and return values in this class with `@NonNull` or `@Nullable` as appropriate?

Reply:
Will tackle this class in a followup